src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -1285,12 +1281,10 @@ public static double ClampAngle(double angle)
         public static float ClampRadians(float angle)
         {
             // mod angle so it's in the range (-2π,2π)
-            angle %= 2 * Pi;
+            angle %= TwoPi;
 
-            // abs angle so it's in the range [0,2π)
-            angle = Abs(angle);
-
-            return angle;
+            // return angle in the range [0,2π)
+            return angle >= 0.0f ? angle : angle + TwoPi;
```

Comment:
I would follow the style of the rest of the functions here and make this an if statement.

Something like
```cs
if (angle < 0.0f)
{
    angle += TwoPi;
}